<ui version="4.0" >
 <class>PackageArchDetailsWidget</class>
 <widget class="QWidget" name="PackageArchDetailsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Packge Details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="archNameGroupBox" >
     <property name="title" >
      <string>Architecture</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLabel" name="remotePackageNameLabel" >
        <property name="text" >
         <string>Remote file name:</string>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="0" column="3" >
       <widget class="QLineEdit" name="remotePackageNameDisplayLineEdit" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="1" column="0" colspan="2" >
       <widget class="QLabel" name="remotePackageSizeLabel" >
        <property name="text" >
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLineEdit" name="remotePackageDisplaySizeLineEdit" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4" >
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="localPackageNameLabel" >
        <property name="text" >
         <string>Local file name:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" >
       <widget class="QLineEdit" name="localPackageNameDisplayLineEdit" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2" >
       <widget class="QLabel" name="localPackageSizeLabel" >
        <property name="text" >
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3" >
       <widget class="QLineEdit" name="localPackageDisplaySizeLineEdit" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
